docs/developer/decision-records/2022-07-06-release-automation/README.md

```diff
@@ -0,0 +1,40 @@
+# Automating the EDC release process
+
+## Decision
+
+We will use GitHub actions to automate the release of both `SNAPSHOT` and release versions to decrease the error
+surface.
+
+A dedicated `releases` branch will be created to host those.
+
+## Rationale
+
+This allows committers to simply trigger the creation of a version through GitHub actions and does not require logging
+in to our JIPP (EF Jenkins) instance and perform manual actions there.
+
+Having a dedicated branch for all releases will it make easier in the future to provide hotfixes for older versions.
+
+## Approach
+
+There will be a new branch called `releases`, which is only used to record the history of our releases, i.e. receive
+merge commits for every release. Then, a new GitHub workflow will be created that:
```

Comment:
Just from reading these lines, I am confused (although I know what we have decided ^^). "Then, a new GitHub..." Does this sentence refer to the one before? If not, just start witih "A new GitHub...".

And it does not get clear how the workflow will be triggered, as this itself triggers a lot of other actions. Does this relate to the "New Release" button press? Why would a workflow provide a promt, as this is provided by the GitHub GUI? This already provides a possibility to insert a semver string and then creates a tag on the chosen branch.

Reply:
> Does this sentence refer to the...

Well, yes it does :) I updated the start of the sentence.

Reply:
> And it does not get clear how the workflow will be triggered, as this itself triggers a lot of other actions. Does this relate to the "New Release" button press? Why would a workflow provide a promt, as this is provided by the GitHub GUI? This already provides a possibility to insert a semver string and then creates a tag on the chosen branch.

The idea is to trigger a workflow, have the Github Website [offer a prompt](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/) where we enter the version. This has nothing to do with _Github Releases_: it would be perfect to somehow piggyback onto GH Releases, and simply take the version from that, but unfortunately that seems impossible at the moment. Therefore we wrap GH Releases into a separate workflow, where it is only one step.

Reply:
okay for me ^^ Maybe it makes sense (as soon as everything is set up) to document the whole process as kind of a "release guide", either in this PR or a markdown, or in our committer team.

Reply:
> okay for me ^^ Maybe it makes sense (as soon as everything is set up) to document the whole process as kind of a "release guide", either in this decision record or a markdown, or in our committer team.

The subsequent PR that introduces the actual workflow will bring that.